rest-backend/src/main/java/com/sap/psr/vulas/backend/repo/GoalExecutionRepositoryImpl.java

```diff
@@ -58,14 +60,14 @@ public GoalExecution customSave(Application _app, GoalExecution _provided_gexe)
 
 		//update the lastScan timestamp of the application (we already have a managed application here)
 		appRepository.refreshLastScanbyApp(_app);
-		
+
 		// Save
 		try {
 			managed_gexe = this.gexeRepository.save(_provided_gexe);
 		} catch (Exception e) {
```

Comment:
We should `sw.stop(e)` in the catch-clause(s)

Reply:
Also note that we will have many such log statements ...

Reply:
Yes